themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Container Service on AWS
```

Comment:
I am just noticing a discrepancy between this and the serverless one. This one says "Container Service on AWS" and the serverless one says "AWS Serverless Application". I think I prefer {Architecture} on {Cloud} format. So we should update the Serverless one to conform

Reply:
the reason the serverless one says "AWS Serverless" is because we are trying to rank on the term "aws serverless" which was identified as an opportunity by 97th floor